CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v2.4.0 - 2023-02-28
+
+Includes:
+
+- Many type annotations added ([#33](https://github.com/octoenergy/xocto/pull/33)).
```

Comment:
@pydanny @codeinthehole does this count as a breaking change (=> major version)? 

It's certainly causing a bit of friction to upgrade here https://github.com/octoenergy/kraken-core/pull/79614#issuecomment-1448085479. Generally if the client has to modify their code to accept the upgrade I would say it should be a major version. But this is just type annotations, and it's only krakens static analysis / linting which breaks. So, I'm not sure.

The removal/renaming of `StorageFile ` is also technically breaking, I would say https://github.com/octoenergy/kraken-core/pull/79614#issuecomment-1447882987. Although the effort for the client to upgrade is small.

Reply:
In Semver breaking changes are a major version upgrades. I think we should do a 3.0 release.